.vimrc

```diff
@@ -68,14 +115,14 @@
 " }
 
 " Use before config if available {
-    if filereadable(expand("~/.vimrc.before"))
-        source ~/.vimrc.before
+    if filereadable(g:spf13_vimrc_before)
+        exec 'source ' . g:spf13_vimrc_before
```

Comment:
Not that I could tell. As far as I know the source command still exec's because it has to source something, but I could be wrong... I spent a little while trying to figure out a way around it but VimL has horrible documentation on the actual language (Vim on the other hand has amazing documentation)